src/common/components/ManageSettingsDropdown.tsx

```diff
@@ -0,0 +1,223 @@
+                        variant="h6-bold"
+                        color="text-primary"
+                        textTransform="capitalize"
+                      >
+                        {setting.user || setting.default}
+                      </Text>
+                      <InputSlider
+                        val={setting.user}
+                        max={setting.upperLimit}
+                        min={setting.lowerLimit}
+                        step={setting.ticker || 1}
+                        defaultVal={setting.default}
+                        onChange={({ x }) => handleSliderChange(index, x)}
+                      />
+                    </Box>
+                  )}
+
+                  {setting.type === 3 && setting.enabled === true && (
+                    <Box
+                      display="flex"
+                      flexDirection="column"
+                      gap="spacing-xxs"
+                      alignItems="flex-start"
+                      alignSelf="stretch"
+                    >
+                      <Text
+                        variant="h6-bold"
+                        color="text-primary"
+                        textTransform="capitalize"
+                      >
+                        {setting.user.lower || setting.default.lower} - {setting.user.upper || setting.default.upper}
+                      </Text>
+                      <RangeSlider
+                        startVal={setting.user.lower || setting.default.lower}
+                        endVal={setting.user.upper || setting.default.upper}
+                        max={setting.upperLimit}
+                        min={setting.lowerLimit}
+                        step={setting.ticker || 1}
+                        defaultStartVal={setting.default.lower}
+                        defaultEndVal={setting.default.upper}
+                        onChange={({ startVal, endVal }) =>
+                          handleSliderChange(index, { lower: startVal, upper: endVal })
+                        }
+                      />
+                    </Box>
+                  )}
+                </Box>
+                <Separator />
+              </Box>
+            );
+          })}
+        </Box>
+
+        <Box
+          display="flex"
+          gap="spacing-xxs"
+          alignSelf="stretch"
+          alignItems="center"
+          justifyContent="flex-end"
+          flexDirection='column'
+        >
+          <Text
+            color="text-tertiary"
+            variant="bes-regular"
+          >
+            You will receive all important updates from this channel.
+          </Text>
+          <Box
+            display='flex'
+            flexDirection='column'
+            gap='spacing-md'
+            alignItems='center'
+            width='100%'
+          >
+            <Button
+              size="small"
+              variant="primary"
+              onClick={handleUpdateNotificationSettings}
+              css={css`width:100%;`}
```

Comment:
```suggestion
block
```
We don't give button this css, use the block prop in button